packs/sensu/etc/st2_handler.conf

```diff
@@ -0,0 +1,5 @@
+---
+st2_username: "testu"
```

Comment:
not a fan of including username and password in config files. I think you should leave both blank.